arch/mips/include/kernel_arch_func.h

```diff
@@ -47,6 +47,8 @@ static inline bool arch_is_in_isr(void)
 	return _current_cpu->nested != 0U;
 }
 
+extern int arch_swap(unsigned int key);
```

Comment:
nit: these `extern` (in all arch's headers) are quite redundant :)

Reply:
Nit exterminated.